Formula/rotz.rb

```diff
@@ -0,0 +1,22 @@
+class Rotz < Formula
+  desc "Fully cross platform dotfile manager and dev environment bootstrapper"
+  homepage "https://volllly.github.io/rotz/"
+  url "https://github.com/volllly/rotz/archive/refs/tags/v0.9.3.tar.gz"
+  sha256 "b7556bdfbee63c9def84a6ce8b12fa41d3f7ee53e2d3b3f240d35110c9ed2f1b"
+  license "MIT"
+  head "https://github.com/volllly/rotz.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--features", "all-formats", *std_cargo_args
+  end
+
+  test do
+    assert_match "rotz #{version}\n", shell_output("#{bin}/rotz -V")
+
+    system bin/"rotz", "clone", "https://github.com/volllly/.dotfiles.git"
```

Comment:
This is probably not desired behavior as it will clone a repo into the user's home without warning. Taking a cue from the reviewer maybe we should assert on the error from running `rotz link`.

```
 $ rotz link
Error:   × Encountered multiple errors

Error: dotfiles::walk

  × Error walking dotfiles
  ├─▶ IO error for operation on /home/mturney/.dotfiles: No such file or directory (os error 2)
  ╰─▶ No such file or directory (os error 2)
```

Reply:
It'll clone into the temporary directory for the test so no issue there.

Reply:
I don't think you are correct. The default behavior for `rotz` is to clone into the current user's home directory and _not_ in a location relative to the current working directory the command is ran from. Unless I am missing something about how `system` calls commands then this will be the current user's HOME and not the temporary location Homebrew runs tests from. Regardless, this repo being cloned here could get very large in the future. There is no reason to download external data when we can just test the output of a command instead.

Reply:
Does it respect `$HOME`? Because that is set to the temporary directory the test runs in.

Reply:
Ah, then yes it should. I am still not a fan of cloning an entire repo just to test if the tool compiled. Seems a bit heavy handed.